containers/images/pulp/container-assets/route_paths.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+import pkgutil
+import json
+import re
+import sys
+import nginx
+from pathlib import Path
+
+nginx_configs = []
+
+for i in pkgutil.iter_modules():
+    # filter pulp and galaxy packages
+    if re.search(r'pulp_.*|galaxy_ng',i.name):
+        # only add the filename to array if it exists
+        if Path(i.module_finder.path + "/" + i.name + "/app/webserver_snippets/nginx.conf").is_file():
+            nginx_configs.append(i.module_finder.path + "/" + i.name + "/app/webserver_snippets/nginx.conf")
+
+name = sys.argv[1]
+router = []
+
+for plugin_conf in nginx_configs:
+    app = "galaxy" if "galaxy" in plugin_conf else plugin_conf.split("pulp_")[1].split("/")[0]
```

Comment:
pulp_conf.split("pulp_")[1].strip("/")

Reply:
```python
In [1]: plugin_conf = "/usr/local/lib/python3.18/site-packages/pulp_container/app/webserver_snippets/nginx.conf"

In [2]: plugin_conf.split("pulp_")[1].strip("/")
Out[2]: 'container/app/webserver_snippets/nginx.conf'
```

Reply:
I believe that in this case, we need to keep using the split("/")[0] to get only the first "string" from the path split by "slashes".
For example, from path:
`/pulp-python/app/webserver_snippets/nginx.conf-pypi`
we'd like to pick only "`pulp-python`".

If we use the .strip("/") we would get a
`pulp-python/app/webserver_snippets/nginx.conf-pypi`
correct? (which is not a valid name for a route path)

Reply:
oh ok, i was using this path `/home/vagrant/devel/pulp_container/pulp_container/app/webserver_snippets/nginx.conf`